cli/src/trades.rs

```diff
@@ -67,3 +67,68 @@ pub fn all(client: &Client, matches: &ArgMatches) -> Result<()> {
     });
     res
 }
+
+pub fn aggregations(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+
+    let endpoint = {
+        let base_asset_type = matches.value_of("base_asset_type").unwrap();
```

Comment:
Prefer using `expect()` so that we can have a specific message of infallibility:

```rust
let base_asset_type = matches.value_of("base_asset_type").expect("Asset type is a required field")
```